templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -18,6 +18,45 @@
     </ul>
     {{- end }}
 
+    <!--
```

Comment:
We should perhaps move this to an issue.

Reply:
It's an issue already (you can see it linked in the comment below). It's high-enough priority that I didn't want us to build upon the code here without pointing out that we need the parent issue fixed.